src/tests/examples/antares_test_utils.py

```diff
@@ -52,18 +52,68 @@ def run_study(solver_path, study_path, use_ortools = False, ortools_solver = "si
     if "Solver returned error" in output[0].decode('utf-8'):
         raise_assertion("Solver returned error")
 
-def enable_study_output(study_path, enable):
-    st = Study(str(study_path))
-    st.check_files_existence()
+# ===============================================================
```

Comment:
I guess we still need this source file. Why do we need it for ?

Reply:
It is no longer needed, I will remove it